src/pages/home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+//requisito 2
```

Comment:
Correção apontada pelo ESLint
```suggestion
// requisito 2
```